app/views/vm_common/_resize.html.haml

```diff
@@ -17,6 +17,11 @@
       .col-md-8
         = select_tag('flavor',
             options_for_select(@flavors.sort),
+            :class             => "selectpicker",
+            "data-miq_sparkle_on" => true, "data-miq_sparkle_off" => true,
             :multiple          => false,
             :id                => "flavor",
             "data-miq_observe" => {:interval => '.5', :url => url}.to_json)
+      :javascript
+        miqInitSelectPicker();
+        miqSelectPickerEvent("flavor", "#{url}");
```

Comment:
@blomquisg Changes in this file fix error raised on screenshots from issue #7899 

Changes in other files fixes problem, that instance resize was not confirmed. It could be tested when #7899 was fixed, so I put it into this PR.